.travis.yml

```diff
@@ -6,6 +6,7 @@ node_js:
 install:
   - npm install
   - npm run build
+  - npm run parcel
```

Comment:
Adding `npm run parcel` is causing travis to fail.

Reply:
@saschanaz  Should I remove this ?

Reply:
No, this is the required step. I think `npm run build` should be completely replaced by `npm run parcel`, as that's the whole purpose of using parcel.